README.elk.md

```diff
@@ -0,0 +1,143 @@
+# Enabling ELK Components for usertracking / analytics
+
+ELK = Elasticsearch + Logstash + Kibana. This is the stack of components used
+for processing and indexing MH analytics data, particularly usertracking events.
+
+* Modify your cluster config to include the following layer configs:
+
+```
+      {
+        "name": "Elasticsearch",
+        "shortname": "elasticsearch",
+        "enable_auto_healing": true,
+        "install_updates_on_boot": true,
+        "type": "custom",
+        "auto_assign_public_ips": false,
+        "use_ebs_optimized_instances": true,
+        "custom_recipes": {
+            "setup": [
+                "mh-opsworks-recipes::set-timezone",
+                "mh-opsworks-recipes::fix-raid-mapping",
+                "mh-opsworks-recipes::set-bash-as-default-shell",
+                "mh-opsworks-recipes::install-utils",
+                "mh-opsworks-recipes::enable-postfix-smarthost",
+                "mh-opsworks-recipes::install-custom-metrics",
+                "mh-opsworks-recipes::create-alerts-from-opsworks-metrics",
+                "mh-opsworks-recipes::enable-enhanced-networking",
+                "mh-opsworks-recipes::install-elasticsearch",
+                "mh-opsworks-recipes::clean-up-package-cache"
+            ],
+            "shutdown": [
+                "mh-opsworks-recipes::remove-alarms"
+            ]
+        },
+        "volume_configurations": [{
+            "mount_point": "/vol/data",
+            "number_of_disks": 1,
+            "size": "50",
```

Comment:
That probably makes sense, as we'd most likely want to use this only for production.